cmd/minishift/cmd/openshift/service.go

```diff
@@ -19,39 +19,25 @@ package openshift
 import (
 	"fmt"
 	"os"
-	"strings"
-	"text/template"
 
-	"github.com/docker/machine/libmachine"
 	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
 
-	"github.com/minishift/minishift/pkg/minikube/cluster"
-	"github.com/minishift/minishift/pkg/minikube/constants"
+	"github.com/minishift/minishift/pkg/minishift/openshift"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 )
 
 var (
-	namespace          string
-	serviceURLMode     bool
-	serviceURLFormat   string
-	serviceURLTemplate *template.Template
-	https              bool
+	namespace string
+	urlMode   bool
+	https     bool
 )
 
 // serviceCmd represents the service command
 var serviceCmd = &cobra.Command{
 	Use:   "service [flags] SERVICE",
-	Short: "Prints the URL for the specified service to the console.",
-	Long:  `Prints the URL for the specified service to the console.`,
-	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		t, err := template.New("serviceURL").Parse(serviceURLFormat)
-		if err != nil {
-			fmt.Fprintln(os.Stderr, "The URL format specified in the --format option is not valid: \n\n", err)
-			atexit.Exit(1)
-		}
-		serviceURLTemplate = t
-	},
+	Short: "Open the URL for the specified service in the browser.",
+	Long:  `Open the URL for the specified service in the browser.`,
```

Comment:
"Opens the URL for the specified service and namespace in the browser or prints it to the console. If no namespace is provided, 'default' is assumed." (something like that)